src/types/Storage/methods/updateAddress.js

```diff
@@ -38,7 +38,7 @@ const updateAddress = function (addressObj, walletId) {
   // if(newObject.utxos.length==0 && previousObject.utxos.length>0){
   //
   // }
-  const currentBlockHeight = this.store.chains[walletStore.network].blockHeight;
+  // const currentBlockHeight = this.store.chains[walletStore.network].blockHeight;
```

Comment:
Is it by mistake or you intended to commit commented code?

Reply:
Intended to remind that we had that to take into account

Reply:
Maybe we need a todo in this case? Just commented code looks dirty https://kentcdodds.com/blog/please-dont-commit-commented-out-code

Reply:
Yes, it's probably true. 
I've read above link, and well 
- Focus and cognitive load : It's important, we lost blockheight, it's degraded state it do compensate cognitive load. 
- Out of date : it's not
- Hides what's important : Does the opposite. 

As we should fix that quite soon enough, it will disappear with that. But right now I need to keep that to re-establish the comportment (and that will be where the diff will help)